kitsune/customercare/forms.py

```diff
@@ -10,21 +11,34 @@ class ZendeskForm(forms.Form):
     """Form for submitting a ticket to Zendesk."""
 
     product = forms.CharField(disabled=True, widget=forms.HiddenInput)
-    category = forms.ChoiceField(
-        label=_lazy("What do you need help with?"), choices=CATEGORY_CHOICES
+    email = forms.EmailField(
+        label=_lazy("Email"), required=False, disabled=True, widget=forms.HiddenInput
     )
+    category = forms.ChoiceField(label=_lazy("What do you need help with?"))
     os = forms.ChoiceField(
         label=_lazy("What operating system does your device use?"),
         choices=OS_CHOICES,
         required=False,
     )
     subject = forms.CharField(label=_lazy("Subject"), required=False)
-    description = forms.CharField(label=_lazy("Your message"), widget=forms.Textarea())
+    description = forms.CharField(label=_lazy("Tell us more"), widget=forms.Textarea())
     country = forms.CharField(widget=forms.HiddenInput)
 
-    def __init__(self, *args, product, **kwargs):
+    def __init__(self, *args, product, user=None, categories=None, **kwargs):
         kwargs.update({"initial": {"product": product.slug}})
         super().__init__(*args, **kwargs)
+        self.fields["category"].choices = categories if categories is not None else []
+        if product.slug in settings.LOGINLESS_PRODUCTS and not user.is_authenticated:
+            self.fields["subject"].initial = "Loginless MA Support"
+            self.fields["subject"].widget = forms.HiddenInput()
+            self.fields["email"].widget = forms.EmailInput()
+            self.fields["email"].disabled = False
```

Comment:
Following the comment above, we can completely omit any manipulation of the email field here

Reply:
Pending meeting....